module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -567,7 +567,179 @@ protected function getSupportedCitationFormats()
      */
     public function exportDisabled($format)
     {
-        // EDS is not export-friendly; disable all formats.
-        return true;
+        // at the moment we only support RIS
+        return strtolower($format) != 'ris';
+    }
+    
+     /**
+     * This method can fetch fields out of the deeply nested RecordInfo array. 
+     * many fields are here in a more readable and atomic format
+     * 
+     * The latter two params are only for internal use (recursion)
+     *  
+     * @param array $arrayKeys Key path to the needed value
+     * @param int $level only used for recursion
+     * @param array $fields only used for recursion
+     * @return array|string
+     */
+    public function getFieldRecursive($arrayKeys, $level = 0, $fields = null) 
```

Comment:
Should this be a protected method? It doesn't seem like it is currently being used outside of the class, and it's probably cleaner to keep it as an internal implementation detail.